Assets/Scripts/BulletManagement/SlipTimeEmitter.cs

```diff
@@ -11,14 +11,100 @@ public class SlipTimeEmitter : EmitterBase, ISlipTimeAdherent
         [SerializeField]
         private SlipTimeManager slipTimeManager;
         
+        // Running total of emitted bullet waves
+        private int cyclesEmitted = 0;
+
+        private bool limited = false;
+
+        public float bulletSpeedMultiplier = 1.0f;
+        
+        // The number of bullet waves to emit. Set to zero for no limit.
+        public int emitCycles = 0;
+
+        // Number of bullets to emit each emit cycle.
+        public int emitBulletCount;
```

Comment:
Looks like Unity is erroring out on my end on the `emitBulletCount` variable which is also in `EmitterBase.cs`.
Since this variable is already in `EmitterBase.cs` probably can be removed in `SlipTimeEmitter.cs` Is it because of rebasing?

Reply:
Yeah, when I was rebasing I misunderstood where Nathaniel had moved things to until I was done with the rebase, so then I tried to shuffle things around, and then meta files got messed up, so I had to remake the branch, so yeah I probably just got this and the other one mixed up when I was remaking things. Odd my compiler didn't error though...